ga4gh/datamodel/variants.py

```diff
@@ -450,9 +450,10 @@ def convertVariant(self, record):
         if record.id is not None:
             variant.names = record.id.split(';')
         variant.start = record.start          # 0-based inclusive
-        variant.end = record.stop             # 0-based exclusive
+        variant.end = record.start + len(record.ref)
```

Comment:
Ah, that makes sense. I was following the GA4GH documentation ("This is typically calculated by start + referenceBases.length") and wondering why my tests were failing. Thanks, I'll fix this.